custom_typings/merge-stream.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "merge-stream" {
+
+  module mergeStream {
+    interface MergedStream extends NodeJS.ReadWriteStream {
+      add: (source: NodeJS.ReadableStream) => MergedStream;
+    }
+	}
```

Comment:
whitespace. this closing curly brace is over indented

related: clang-format? automatically eliminate almost all alignment angst!

see polymer-analyzer for how to set it up, it's very easy and very nice